ValueChangedGenerator/ValueChangedGenerator/ValueChangedGenerator.Test/Verifiers/CSharpAnalyzerVerifier`1+Test.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Testing;
```

Comment:
These files in Verifiers folder can be removed. There exist equivalent codes in the Assembly Microsoft.CodeAnalysis.Analyzer.Testing and Assembly Microsoft.CodeAnalysis.CodeFix.Testing packages.